Source/Scene/Cesium3DTileset.js

```diff
@@ -876,6 +958,32 @@ define([
             get : function() {
                 return this._statistics;
             }
+        },
+
+        /**
+         * @private
+         */
+        skipSSEFactor : {
+            get : function() {
```

Comment:
This should return the value passed in.

The setter can set a `_computedSkipSSEFactor` for internal use in this class.